app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterPresenter.kt

```diff
@@ -1,19 +1,20 @@
 package com.puzzlebench.clean_marvel_kotlin.presentation.mvp
 
 import com.puzzlebench.clean_marvel_kotlin.domain.usecase.GetCharacterServiceUseCase
+import com.puzzlebench.clean_marvel_kotlin.domain.usecase.GetCharacterStoreUseCase
 import com.puzzlebench.clean_marvel_kotlin.presentation.base.Presenter
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.schedulers.Schedulers
 
-class CharacterPresenter(view: CharecterView, private val getChatacterServiceUseCase: GetCharacterServiceUseCase, val subscriptions: CompositeDisposable) : Presenter<CharecterView>(view) {
+class CharacterPresenter(view: CharecterView, private val getChatacterServiceUseCase: GetCharacterServiceUseCase, private val storeCharacterServiceUseCase: GetCharacterStoreUseCase, val subscriptions: CompositeDisposable) : Presenter<CharecterView>(view) {
```

Comment:
your presenter constructor should be like this:

class CharacterPresenter(view: CharecterView, model CharacterPresenterModel, val subscriptions: CompositeDisposable) : Presenter<CharecterView>(view)

instead of passing all the useCases.